esmvaltool/diag_scripts/surface_trace_gases/diagnostic_surface_trace_gases.py

```diff
@@ -0,0 +1,2396 @@
+"""Surface concentration of trace gases diagnostics.
+
+The diagnostics rely on comparing model data to ground-based NOAA GML surface
+observations of trace gases.
+
+Some classes and functions largely adapted from the AOD-AERONET diagnostic
+at /diag_scripts/aerosols/aod_aeronet_assess.py.
+"""
+
+import logging
+from pathlib import Path
+
+import cartopy.crs as ccrs
+import iris
+import iris.analysis
+import iris.coords
+import iris.plot as iplt
+import matplotlib as mpl
+import matplotlib.lines as mlines
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy
+from esmvalcore.preprocessor._time import climate_statistics
```

Comment:
```suggestion
from esmvalcore.preprocessor import climate_statistics
```
Objects starting with an underscore should be considered private and not used outside the module they are defined in according to [PEP8](https://peps.python.org/pep-0008/), so please avoid using those as these may change from one minor version to the next without warning.